server/command.go

```diff
@@ -173,8 +173,14 @@ func executeInstanceAdd(p *Plugin, c *plugin.Context, args ...string) *model.Com
 		return responsef(addResponseFormat, ji.GetURL(), p.GetSiteURL(), ji.GetMattermostKey(), pkey)
 
 	case JIRATypeCloud:
+		// Create an "uninitialized" instance of Jira Cloud that will
+		// receive the /installed callback
+		err := p.StoreNewCloudInstance(jiraURL)
+		if err != nil {
+			return responsef(err.Error())
+		}
 		// TODO the exact group membership in Jira?
-		return responsef(`As an admin, upload an application from %s/%s. The link can be found in **Jira Settings > Applications > Manage**`,
+		return responsef(`As an admin, upload an application from %s/%s (expires in 15 minutes). The link can be found in **Jira Settings > Applications > Manage**`,
```

Comment:
Why 15 minutes? (instead of shorter/longer time)

Reply:
No particular reason, seems sufficient. Would you like me to change this or make it configurable?

Reply:
15 minutes feels a bit short. I'd make it as long as is reasonable security-wise.

If it's 15 minutes, someone starting to set up a Jira integration before their meeting, holds their meeting, then continues after their meeting will result in an error they may not immediately understand why. It's not forgiving for the admin.

Reply:
@crspeller do you have a suggestion on what is "as long as is reasonable security-wise"?